debian/neofs-node.docs

```diff
@@ -0,0 +1,5 @@
+docs/*
```

Comment:
Do we need [release-instruction.md](https://github.com/nspcc-dev/neofs-node/blob/master/docs/release-instruction.md) and [update-go-instruction.md](https://github.com/nspcc-dev/neofs-node/blob/master/docs/update-go-instruction.md)?

Reply:
... or we'll have to update "docs" file in case of changing docs/ content